guppylang/prelude/builtins.py

```diff
@@ -74,7 +74,14 @@ def __rmatmul__(self, other: Any) -> Any:
         return other
 
 
-inout = _Inout()
```

Comment:
We should also remove the `_Inout` class

Reply:
Removed